canister/src/lib.rs

```diff
@@ -210,6 +217,26 @@ fn verify_api_access() {
     });
 }
 
+/// Verifies that if the difference between the maximum height
+/// of all block headers and the maximum height of all unstable
+/// blocks is at most the SYNCED_THRESHOLD.
+fn verify_fully_synced() {
```

Comment:
Given that we allow a gap, wouldn't it be better to call it `verify_synced`?

Reply:
Done.